src/books/books.service.ts

```
@@ -1,120 +1,105 @@
+                    bookIds: [book._id.toString()],
+                });
+            }
+        }
+
+        return book;
     }
 
-    return book;
-  }
+    async updateBook(id: string, params: UpdateBookInput): Promise<BookDocument> {
+        const authorIds = params.authorIds && params.authorIds.map(id => new Types.ObjectId(id));
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)